### PR TITLE
update summit profile

### PR DESCRIPTION
**Pull request by psychocoderHPC**
Update used modules for summit ORNL.

### CODE REVIEW

**Review by sbastrakov — DISMISSED** [review]

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 1]
```diff
@@ -18,19 +18,19 @@ export proj=<yourProject>
...
+module load cmake/3.18.2
+module load cuda/10.1.243
 module load boost/1.66.0
 
 # plugins (optional) ##########################################################
-module load hdf5/1.10.3
+module load hdf5/1.10.4
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 module load ums
 module load ums-aph114
```
Comment (PrometheusPi) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example:36]: loading this module causes the `zfp` module to change:
```
The following have been reloaded with a version change:
  1) zfp/0.5.2 => zfp/0.5.5
  ```
  Perhaps we should use `zfp/0.5.5` from the start. @psychocoderHPC what do you think?

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 2]
```diff
@@ -18,19 +18,19 @@ export proj=<yourProject>
 #export EDITOR="nano"
 
 # basic environment ###########################################################
-module load gcc/6.4.0
+module load gcc/8.1.1
```
Comment (PrometheusPi) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example:21]: gives the warning:
```

Lmod is automatically replacing "xl/16.1.1-5" with "gcc/8.1.1".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/10.3.1.2-20200121
  ```

Should be fine.

**Review by PrometheusPi — DISMISSED** [review]
Looks good - we could avoid one warning.

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example (hunk 2 above)
Comment (psychocoderHPC) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example:21]: Yes this is fine

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example (hunk 1 above)
Comment (psychocoderHPC) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example:36]: I did it first and running into an issue, I do not remember which. I can try again.

Comment (psychocoderHPC) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example:34]: I fixed it by loading `ums` first, this is what `spider` said to me.

**Comment by psychocoderHPC** [comment]
ping

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example (hunk 1 above)
Comment (PrometheusPi) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example:34]: works perfectly :+1:

**Review by PrometheusPi — APPROVED** [review]

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 3]
```diff
@@ -18,22 +18,22 @@ export proj=<yourProject>
...
```
Comment (ax3l) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example:21]: Brave of you, GCC 8.0-8.3 is very bug riddled (8.4+ is fine again) :-)

Comment (psychocoderHPC) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example:21]: Thanks, I was switching to the newest compiler which is supported by CUDA 10.2 and provides all modules we need. I will have a look if 8.4+ is available with all required modules.

**Pull request merged**